src/main/java/at/hannibal2/skyhanni/config/features/Garden.java

```diff
@@ -1030,4 +1030,12 @@ public static class CustomSpeed {
 
     @Expose
     public Position cropSpeedMeterPos = new Position(278, -236, false, true);
+
+    @Expose
+    @ConfigOption(name = "Yaw / Pitch", desc = "Show Yaw and Pitch")
```

Comment:
Please add a point at the end of the description.

Reply:
Second time I forgot about the dot. Added and will push it after resolving the other review.

Reply:
done